api/scpca_portal/templates/readme/metadata_only.md

```diff
@@ -0,0 +1,55 @@
+
+Generated on: {{ date }}
+
+# Alex's Lemonade Stand Foundation Single-cell Pediatric Cancer Atlas
+
+The [Single-cell Pediatric Cancer Atlas](https://scpca.alexslemonade.org) is a database of single-cell and single-nuclei data from pediatric cancer clinical samples and xenografts, built by the [Childhood Cancer Data Lab](https://www.ccdatalab.org/) at [Alex's Lemonade Stand Foundation](https://www.alexslemonade.org/).
+
+## Contents
+
+{% if project_accession %}
+This download includes associated metadata for samples from project [{{ project_accession }}]({{ project_url }}) in the ScPCA portal.
+{% else %}
+This download includes associated metadata for samples from all projects currently available at time of download in the ScPCA portal.
+{% endif %}
+
```

Comment:
```suggestion
The metadata included in this download contains sample, library, and project-related metadata (e.g., age, sex, diagnosis, sequencing unit, etc.) along with any relevant processing metadata (e.g., software versions, filtering methods used, etc.). 
```
Probably useful to expand on the contents of what they can expect in the metadata slightly although the full description is in the docs link.

Reply:
Great! Added.